src/backend/utils/metaHotTeardown.ts

```diff
@@ -5,9 +5,15 @@
 // teardown the side effect that main() has.
 // https://github.com/vitest-dev/vitest/issues/2334
 
-export const metaHotTeardown = (teardown: () => unknown) => {
-  if (import.meta.hot) {
```

Comment:
`import.meta.hot` is scoped to this file.  It's not -this- file `src/backend/utils/metaHotTeardown.ts`.  So as it was it was waiting for the metaHotTeardown.ts file to be reloaded then it'd call the server.close() cleanup in main.

But that's not what I want!  Instead I will pass the import.meta.hot from the file that is causing the side effect.